### DISCUSSION TITLE
Welcome to libzetta-rs Discussions!

### DISCUSSION THREAD

**Discussion by andoriyu** [Inner-Heaven/libzetta-rs#157] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andoriyu** [comment]
@unrelentingtech once your patch gets merged and release in FreeBSD, would you mind pinging me so I can fix CI on freebsd?

**Reply by valpackett** [reply]
yea of course

**Reply by andoriyu** [reply]
@unrelentingtech is it still in review?

**Reply by valpackett** [reply]
The dependency one? No, that has landed some time ago: https://reviews.freebsd.org/R10:9e9c651caceb9ecd17131e8bb29791ba4cf1cec7
